lib/Parser/RegexCompileTime.cpp

```diff
@@ -3460,6 +3460,12 @@ namespace UnifiedRegex
         innerFollow->UnionInPlace(compiler.ctAllocator, *accumFollow);
         innerFollow->UnionInPlace(compiler.ctAllocator, *body->firstSet);
 
+        if (followSet->IsSingleton())
```

Comment:
If it is singleton, it means that it has 1 element.

---
In reply to: [81023296](https://github.com/Microsoft/ChakraCore/pull/1579#discussion_r81023296) [](ancestors = 81023296,81023203)